docs/editors-guide/quality-control-tests.md

```diff
@@ -1000,6 +1003,31 @@ ORDER BY ?entity
 
 ```
 
+###  qc-excluded-subclass.sparql
+
+```
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+
+# Tests if an animal disease made it into the rare disease subset
```

Comment:
@matentzn how does this test if an animal disease made it into the rare disease subset?

Reply:
This is a wrong comment copy-pasted falsely from a different test, needs to be changed in src/sparql/qc/*/qc-excluded-subclass.sparql